spec/bowlingScoreSpec.js

```diff
@@ -0,0 +1,117 @@
+describe('BowlingScore', function() {
+  var game = new BowlingScore();
```

Comment:
This should be wrapped into a `before` block, as right now you're creating a state dependency between your tests. For example if you move one test out of order it will break another one. You want each test to stand as an individual statement about what your application can do